src/main/java/soot/dexpler/DexAnnotation.java

```diff
@@ -483,25 +505,17 @@ public void add(AnnotationTag a, int visibility) {
           clazz.setOuterClass(SootResolver.v().makeClassRef(outerClass));
           assert clazz.getOuterClass() != clazz;
         }
-
-        // EnclosingClass comes in pair with InnerClass.
-        // Those are generated from a single InnerClassTag,
-        // that is re-constructed only for the InnerClass Dalvik
-        // annotation.
-        continue;
-
-      } else if (atypes.equals("dalvik.annotation.EnclosingMethod")) {
+        return;
```

Comment:
Sometimes cases end with "break", sometimes with "return". Can we have a single concept here?

Reply:
Just comment the return with "do not add tag" or the like, that would be sufficient